go.mod

```diff
@@ -1,14 +1,16 @@
 module github.com/containers/skopeo
 
 // Minimum required golang version
-go 1.23.0
+go 1.23.3
+
+toolchain go1.23.9
```

Comment:
I think downstreams would prefer this line to be dropped.

(Even the 1.23.3 requirement causes build failures, I assume that has also happened for Podman and we have a plan for that.)

Reply:
good thought, let me go tweak that.